### PR TITLE
Prepare 1.0 release

### PR DESCRIPTION
**Pull request by vhf**
Note: tests won't run until `rdf-loader-code` has moved to ESM

### CODE REVIEW

File: test/fetch.test.js [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
+import { describe, it, beforeEach, before, after } from 'mocha'
+import chai from 'chai';
+import { create } from 'barnard59/lib/runner.js'
+import { resolve, dirname } from 'path'
+import clownface from 'clownface-io'
+import rdf from '@rdfjs/data-model';
+import toString from 'stream-to-string'
+import toStream from 'string-to-stream'
+import sinon from 'sinon';
+import proto, { fetch } from '../protoFetch.js'
```
Comment (tpluscode) [inline test/fetch.test.js:10]: I don't think you can restructure a default export like that.

I'd rather export named and import like
```suggestion
import * as proto from '../protoFetch.js'
```

Comment (vhf) [reply test/fetch.test.js:10]: Sure you can, [see](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#syntax): `import defaultExport, { export1 [ , [...] ] } from "module-name";`

Comment (tpluscode) [reply test/fetch.test.js:10]: I'm pretty sure this only works when `export1` is a named export.

Comment (vhf) [reply test/fetch.test.js:10]: Yep and `export1` is a named export.

Comment (tpluscode) [reply test/fetch.test.js:10]: Oh, ok, `fetch` is exported both named and as `default { fetch }`. Why have it like that?

File: index.js [hunk 2]
```diff
@@ -1,5 +1,3 @@
-const fetchCsv = require('./lib/fetch')
+import fetchCsv from './fetch.js'
```
Comment (tpluscode) [inline index.js:1]: I'd just reexport, like

```suggestion
export { fetchCsv as fetch } from './fetch.js'
```

Comment (vhf) [reply index.js:1]: I think you mean `export { default as fetch } from './fetch.js'`?

Comment (tpluscode) [reply index.js:1]: right, right

**Pull request closed without merging**